serverless.yml

```diff
@@ -33,20 +33,13 @@ provider:
       Resource: 'arn:aws:logs:*:*:*'
   environment:
     hammerTimeDryRun: '${file(./isDryRun.js):isDryRun}'
-  custom:
-    enabled: '${file(./isEnabled.js):isEnabled}'
+    defaultOperatingTimezone: '${file(./operatingTimezones.js):defaultOperatingTimezone}'
 functions:
   stop-hammertime:
     handler: hammertime.stop
     timeout: 180
```

Comment:
We need a longer timeout these days, could be subsequent PR though to make it a parameter :)